webclient/gulpfile.js

```diff
@@ -638,6 +640,26 @@ function copyMapJS() {
 }
 gulp.task("map", gulp.parallel(copyMapCSS, copyMapJS));
 
+// --- api-visualiser (currently swagger-ui) Pipeline ---
+function copyApiCSS() {
+  return gulp
+    .src("node_modules/SwaggerDark/SwaggerDark.css")
+    .pipe(postcss([
+      postcssPrependSelector({ selector: "body.theme-dark " })
```

Comment:
```suggestion
      postcssPrependSelector({ selector: "body.theme-dark #swagger-ui " })
```
Alternatively, this works:
![image](https://user-images.githubusercontent.com/26258709/184555608-153be0bf-6e64-43dd-88f2-25cceba80a24.png)
![image](https://user-images.githubusercontent.com/26258709/184555621-331d209e-1549-4149-aa48-673123e7bada.png)

Reply:
This is good in any case I think